### DISCUSSION TITLE
some workers seem to have died and gunicorn did not restart them as expected

### DISCUSSION THREAD

**Discussion by wxmgmh** [apache/airflow#32336] [Q&A]
The worker, webserver, and scheduler of Airflow started normally, but the webserver log reported an error of
ERROR - [0 / 0] some workers see to have died and Gunicorn did not restart them as expected
INFO - Received singal:15.closing gunicorn.

**Comment by potiuk** [comment]
Then  you should look at your deployment logs. Likely you have resource problems (memory?) if you can't find logs in airflow but you should find them in the lofgos of whatever deployment you chose (and you have not mentioned how you deployed airflow). 

But only you can tell what deployment it is and check the logs there.
